src/luaconnector/connection.h

```diff
@@ -0,0 +1,77 @@
+#ifndef _CONNECTION_H
+#define _CONNECTION_H
```

Comment:
This has to be prefixed to avoid collisions.
Could also switch to `#pragme once`, which would be proper WebKit style
```suggestion
#ifndef _LUACONNECTOR_CONNECTION_H
#define _LUACONNECTOR_CONNECTION_H
```